internal/format/templates/pretty.tmpl

```diff
@@ -30,19 +30,24 @@
         {{- range . }}
             {{- printf "module.%s" .Name | colorize "\033[36m" }}{{ printf " (%s)" .FullName }}
         {{ end -}}
-        {{- printf "\n\n" -}}
     {{ end -}}
+    {{- printf "\n\n" -}}
```

Comment:
Was there a reason for this change? I think it's not needed.

Reply:
you're right, removing two `-` from previous  `{{ end }}` I got the same result!